README.md

```diff
@@ -42,7 +42,8 @@ defmodule MyApp.SomeServiceBridge do
     http_options: [timeout: 1_000, recv_timeout: 16_000],
     max_attempts: 1,
     encode_variables: false,
-    format_response: true # formats keys from CamelCase to snake_case
+    format_response: true, # formats keys from CamelCase to snake_case
+    log_options: [log_query_on_error: true]
```

Comment:
I'd suggest to also add eg. `log_response_on_error: false` to the example in README, so that all possible options are documented in the README (until we write more proper documentation for the library 🙂 )